0000-relationship-between-protocols/0000-relationship-between-protocols.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Relationship between Protocols
+draft: 1
+---
+
+# Relationship between Protocols
+
+## Prerequisites
+This document assumes the reader is familiar with the following document:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+
+## Terminology
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
+- A **payment** is sending value from one to another but it doesn't mean settled necessarily that is explained below.
+- A **settlement** is ensuring that one has the paid value. For instance, executing a transaction on a certain ledger that the payee has an account on.
+
+## Motivation
+Because Interledger Protocol consists of several protocols and some related specifications, it is slightly difficult to grasp the entire relationship between the protocols. It is important for those who concretely implement ILP, to explicate the relationship so that they could quickly imagine how they should go forward. This document aims to build their mental models for ILP. A mental model is what a reader assumes something to be, reading some explanation. The mental model helps the reader predicting the behavior of what is explained even if the details are not given at that time.
+
+## Scope
+The scope of this document is the explication of the relationship between the Interledger protocols, and the details for it are out of scope. Some references are given for it though. i.e. This document draws a whole, big picture of ILP.
+
+## Perspective
+First of all, the diagram below shows the entire relationship between the protocols.
+
+![Perspective](images/perspective.svg)
+
+### Elements and Brief Explanations
+
+- Connections
+  - (A) BTP over WebSocket
+    - **Node** to **Node** connection
+    - In order to exchange payments, configuration and routing information, a node MUST have a WebSocket connection to another node.
+  - (B) Ledger specific connection
+    - **Node** to **Ledger** connection
+    - A node needs the means to settle its payments. Therefore the node has a connection to a ledger. The type of connection varies depending on the ledger.
+  - (C) SPSP over HTTPS
+    - **Application** to **SPSP Server** connection
+    - To determine payment information such as a shared secret, a destination address and so forth, an application connects to SPSP server over HTTPS.
+- Protocols and related specifications
+  - (1) BTP
+    - [Bilateral Transfer Protocol 2.0 (BTP/2.0)](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md)
+    - BTP is used for transferring ILP packets and messages that are used for settlements and so on between two nodes.
+  - (2) ILP
+    - [Interledger Protocol V4 (ILPv4)](../0027-interledger-protocol-4/0027-interledger-protocol-4.md)
+    - ILP is used for transferring payment, node configuration, and routing information.
```

Comment:
I might say something like "ILP is used for sending payment packets across multiple hops. Some other protocols, including those for node configuration and routing, are also built on top of ILP and use ILP packets to communicate that information between peers."